modules/nf-core/ensemblvep/vep/main.nf

```diff
@@ -62,7 +62,7 @@ process ENSEMBLVEP_VEP {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     def file_extension = args.contains("--vcf") ? 'vcf' : args.contains("--json")? 'json' : args.contains("--tab")? 'tab' : 'vcf'
-    def create_index = file_extension == "vcf" ? "touch ${prefix}.${file_extension}.gz" : ""
+    def create_index = file_extension == "vcf" ? "touch ${prefix}.${file_extension}.gz.tbi" : ""
```

Comment:
Shouldn't the `.tbi` be before the `.gz`??
```suggestion
    def create_index = file_extension == "vcf" ? "touch ${prefix}.${file_extension}.tbi.gz" : ""
```

Reply:
No the `tbi` is behind it :p

Reply:
It's the index of the `.vcf.gz` file :p

Reply:
Uh ok :sweat_smile: